src/extends/room/meta.js

```diff
@@ -164,6 +164,12 @@ Room.isHallway = function (name) {
 }
 
 Room.isClaimable = function (name) {
+  if (Game.gcl.level >= GCL_NOVICE && Game.map.getRoomStatus(name).status !== 'normal') {
```

Comment:
I didn't see that either yesterday: We should check for owned rooms instead of current GCL.